### DISCUSSION TITLE
How to use sparkle with multiple apps on same device

### DISCUSSION THREAD

**Discussion by Beingpax** [sparkle-project/Sparkle#2651] [Q&A]
I tried using the same old key for second app as well, but it doesn't generate sparkle:edSignature. I tried to remove the key from the keychain and created a new one and added to the new app. But now I can't do the same for the second app. I think I need to again delete this new key and use the old key maybe importing it again. 

Is there a way to handle this?

A detailed response would be very helpful.

**Comment by zorgiepoo** [comment]
The same key can be used for multiple apps. What commands are you trying and what output / errors are you running into. You can use another key if you pass ‘--account’ to the CLI tools for generating and signing updates but it’s not required. 

There is likely some other issue, like you didn't specify the public key in your app's Info.plist correctly, for example.
